frontend/src/components/Game.js

```diff
@@ -1,13 +1,198 @@
+            setDirectories(subdirectories);
+          })
+          .catch((e) => {
+            const message = "Error occurred: " + e.message;
+            console.log(message);
+            setError(message);
+          });
+      })
+      .catch((e) => {
+        const message = "Error occurred: " + e.message;
+        console.log(message);
+        setError(message);
+      });
+  }, [gameId]);
+
+  const handleClickChoose = (directoryName) => {
+    console.log(`Chose directory: ${directoryName}`);
+    setAnswer(directoryName);
+  };
+
+  const handleSubmit = (answer) => {
+    if (answer) {
+      const options = {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          answer: answer,
+        }),
+      };
+
+      fetch(`${BACKEND_URL}/game/${gameId}`, options)
+        .then((response) => {
+          if (response.ok) {
+            setAnswerSubmitted(true);
+            console.log("Submitted successfully");
+          } else {
+            throw new Error("Failed");
+          }
+        })
+        .catch((error) => {
+          console.log("Error occurred:", error.message);
+        });
+    }
+  };
+
+  const handleReturn = () => {
+    if (pathHistory.length > 0) {
+      const previousPathEntry = pathHistory[pathHistory.length - 1];
+      setDirectories(previousPathEntry.directories);
+      setCurrentPath(previousPathEntry.path);
+      setPathHistory((prevPathHistory) =>
+        prevPathHistory.slice(0, prevPathHistory.length - 1)
+      );
+    }
+  };
 
   return (
     <>
-      <h1>Game</h1>
-      <div>Game id = {state.gameId}</div>
+      <h1 className="title">gitguesser</h1>
+      <div className="form">
+        <div className="info">
+          Player: {playerName}
+          <br />
+          Current path: {currentPath}
+        </div>
+        <h2 className="directories">Directories:</h2>
+        <button className="backButton" onClick={handleReturn}>
+          Back
+        </button>
+        <ul>
+          {directories.map((directory) => (
+            <li key={directory.id}>
+              <button
+                className="buttonDir"
+                onClick={() =>
+                  handleClickDirectory(
+                    directory.id,
+                    repositoryId,
+                    directory.name
+                  )
+                }
+              >
+                {directory.name}
+              </button>
+              <button
+                className="buttonChoose"
+                onClick={() => handleClickChoose(directory.name)}
+              >
+                Choose
+              </button>
+            </li>
+          ))}
+        </ul>
+        {answer && <div className="chosenDir">Chosen directory: {answer}</div>}
+        <button
+          type="submit"
+          className="buttonSubmit"
+          onClick={() => handleSubmit(answer)}
```

Comment:
Can you make this button disabled when the player hasn't chosen any directory yet?